src/olympia/editors/views.py

```diff
@@ -650,6 +651,14 @@ def review(request, addon, channel=None):
         except AddonApprovalsCounter.DoesNotExist:
             pass
 
+    reports = None
+    user_reviews = None
+    if addon.has_listed_versions():
```

Comment:
https://github.com/mozilla/addons-server/pull/5315/commits/d6dbed1125caac2ff61933ef2fe0660096d9db62 uses the `channel`, as well as only displaying all that stuff if the current version was auto approved.